bazel/cargo/wasmsign/Cargo.toml

```diff
@@ -8,13 +8,3 @@ path = "fake_lib.rs"
 
 [dependencies]
 wasmsign = {git = "https://github.com/jedisct1/wasmsign", revision = "fa4d5598f778390df09be94232972b5b865a56b8"}
-
-[package.metadata.raze]
-rust_rules_workspace_name = "rules_rust"
-gen_workspace_prefix = "wasmsign"
-genmode = "Remote"
-package_aliases_dir = "."
-workspace_path = "//bazel/cargo/wasmsign"
-
-[package.metadata.raze.crates.wasmsign.'*']
-extra_aliased_targets = ["cargo_bin_wasmsign"]
```

Comment:
Maybe the wasmsign issue is related to the removal of this line, without something to achieve the same with crate_universe? The relevant (defunct) cargo-raze documentation appears to be [this](https://github.com/google/cargo-raze?tab=readme-ov-file#crates-that-supply-useful-binaries).

crate_universe also supports `extra_aliased_targets`, documentation [here](https://bazelbuild.github.io/rules_rust/crate_universe.html#crate.annotation-extra_aliased_targets).

Reply:
Thanks! Fixed in the latest commit via `generate_binaries = True`.